packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -20,24 +20,47 @@ The `core`, `support`, `layout` npm packages have been deprecated and replaced w
 
 ```
 
-## Importing JavaScript and Sass
+## Folder structure
 
-Moving forward in v2, we will be distributing `sass` files in the `dist` directory in the `@cmsgov/design-system` npm package. If you were importing JavaScript or Sass directly from our packages’ `src` directories, it's recommended you update your import paths to import from `dist`.
+The `dist` folder structure has been updated for the `@cmsgov/design-system` npm package. We recommend only importing files from the `dist` directory moving forward.
 
-For JavaScript imports, this will likely result in no changes. While JavaScript files were always available in the `src` directory before, it was never recommended to import the source version directly, so your project likely will not contain any `src` folder imports for React components.
+```
+└── @cmsgov/design-system/dist
+    ├── components/
+    │   └── index.js      Compiled JS (CommmonJS)
+    ├── css/
+    │   └── index.css     Compiled CSS
+    ├── esnext/
+    │   └── index.esm.js  Compiled JS (ES Module)
+    ├── fonts/
+    ├── images/
+    └── scss/             Uncompiled CSS
+```
+
+### Importing SCSS/CSS
+
+If you were importing Sass directly from our packages’ `src` directories, we recommended to update your import paths to import from `dist`.
```

Comment:
change: 
from: we recommended to update your import paths to import from `dist`
to: we recommend updating your import paths to import from `dist`.